README.md

```diff
@@ -343,5 +352,33 @@ The generated report will be available at ```target/site/jacoco/index.html```
+| `backupFile`                               | The path to the file where [local backups](#local-backup) get stored.                                                                                               | No       | `"java.io.tmpdir/unleash-repo.json"` |
+| `customHttpHeader`                         | Add a [custom HTTP header](#custom-http-headers) to the list of HTTP headers that will the client sends to the Unleash API. Each method call will add a new header. | No       | N/A                                  |
+| `customHttpHeadersProvider`                | Add a custom HTTP header provider. Useful for [dynamic custom HTTP headers](#dynamic-custom-http-headers).                                                          | No       | `null`                               |
+| `disableMetrics`                           | A boolean indicating whether the client should disable sending usage metrics to the Unleash server.                                                                 | No       | `false`                              |
+| `enableProxyAuthenticationByJvmProperties` | Enable support for [using JVM properties for HTTP proxy authentication](#http-proxy-with-authentication).                                                           | No       | `false`                              |
+| `environment`                              | The name of the current environment.                                                                                                                                | No       | `null`                               |
+| `fallbackStrategy`                         | ???                                                                                                                                                                 | No       | `null`                               |
+| `fetchTogglesInterval`                     | How often (in seconds) the client should check for toggle updates.                                                                                                  | No       | `10`                                 |
+| `instanceId`                               | An identifier for your instance. Typically a hostname, pod id or something similar. Unleash uses this internally for ...something                                   | Yes      | `null`                               |
+| `namePrefix`                               | ???                                                                                                                                                                 | No       |                                      |
+| `projectName`                              | The name of the Unleash project?                                                                                                                                    | No       |                                      |
```

Comment:
This probably isn't the Unleash project name, but instead ... something Java-related?

Reply:
It is the Unleash project name! Related to the one above, this nudges Unleash to only query toggles in the given project. You should be able to do the same thing with an appropriate token constrained to only that project (this might be if you want to turn off Auth?)